### ISSUE TITLE
Extension causing "return statement is missing" false positives

### ISSUE THREAD

**Issue by xificurk** [pepakriz/phpstan-exception-rules#113]
After update to PHPStan 0.12 and PHPStan exception rules 0.10 we started getting a lot of false positives of type `Method *** should return *** but return statement is missing.`

It looks like the root cause is in this extension. Here is a minimal code to reproduce the error:

```
<?php declare(strict_types = 1);

trait FooTrait
{
	abstract public function getString() : string;
}

class Foo
{
	use FooTrait;
	public function getString() : string
	{
		return 'Foo';
	}
}

```

With bare phpstan no errors are reported.
After enabling exception rules extension the following error is reported:
```
------ ------------------------------------------------------------------------------- 
  Line   Foo.php (in context of class Foo)                                              
 ------ ------------------------------------------------------------------------------- 
  5      Method Foo::getString() should return string but return statement is missing.  
 ------ ------------------------------------------------------------------------------- 

 [ERROR] Found 1 error
```

**Comment by pepakriz** [comment]
Hi @xificurk . Thanks for your report. This PR should work https://github.com/pepakriz/phpstan-exception-rules/pull/114
Can you test it and confirm it?

**Comment by xificurk** [comment]
Thanks for fast reply. I can confirm the fix #114 works on our code base.
